src/objects/bio/bio.scss

```diff
@@ -14,18 +14,27 @@
     'avatar'
     'content'
     'meta';
-  grid-template-rows: repeat(3, minmax(0, auto));
 
   @media (min-width: breakpoint.$l) {
+    gap: 1em;
     grid-template-areas:
       'avatar content content'
       'meta content content';
     grid-template-columns: repeat(3, 1fr);
   }
 }
 
+///
+/// 1. Match prototype designs
+///
+
 .o-bio__avatar {
   grid-area: avatar;
+  inline-size: 12em; // 1
+  margin: 0 auto;
+  @media (min-width: breakpoint.$l) {
+    inline-size: 14em; // 1
```

Comment:
Instead of having this grow by an arbitrary amount, could we increase the `font-size` of this element to the second scale step? That way it'll be based on our modular scale.

Reply:
Thanks, @tylersticka! I understand the words you've typed. But I don't understand how to implement that. Is there an example I can reference?  🤔

Reply:
You can use the `ms.step` mixin: https://github.com/cloudfour/cloudfour.com-patterns/blob/ab373f869babc497ce2394ca0995fd387ed00d50/src/mixins/_ms.scss#L4-L6

```scss
@include ms.step(1);
```

This will result in:

```css
font-size: 1.25em;
```

Which should cause the width or max-width value to increase proportionally.

Reply:
[Did a thing](https://github.com/cloudfour/cloudfour.com-patterns/pull/1747/commits/63a608784b2140e725906611f646964fe850a506)! 🙂